content/en/continuous_delivery/deployments/ciproviders.md

```diff
@@ -123,6 +123,10 @@ the `DD_GITHUB_JOB_NAME` environment variable needs to be exposed, pointing to t
         - run: datadog-ci deployment mark ...
     ```
 
+## Correlate deployments with CI pipelines
```

Comment:
```suggestion
## Visualize correlated pipelines and deployments
```

Reply:
Suggesting to reword this section to more accurately describe its content. However, I'm open to feedback on this as well.

Reply:
Makes sense to me as there is no additional action needed for the customer to make it happen in this case